src/parser.rs

```diff
@@ -122,6 +122,7 @@ fn decl_name(ctx: &mut ClangParserCtx, cursor: &Cursor) -> Global {
             CXCursor_ClassTemplate |
             CXCursor_ClassDecl |
             CXCursor_StructDecl => {
+                let anno = Annotations::new(&cursor);
```

Comment:
That's visit_top, which AFAICT is only called for toplevel decls. decl_name otoh is called for all decls